modules/log-collection-rbac-permissions.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-multi-clf.adoc
+
+:_content-type: PROCEDURE
+[id="log-collection-rbac-permissions_{context}"]
+= Authorizing log collection RBAC permissions
+
+In logging 5.8 and newer versions, the Cluster Logging Operator provides `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles, which correspond to roles that are able to collect audit logs, application logs, and infrastructure logs respectively.
+
+You can authorize RBAC permissions for log collection by binding the required cluster roles to a service account.
+
+[NOTE]
+====
+Cluster role binding is not required in legacy versions of logging (5.7 and older).
```

Comment:
It is required for what the enhancement calls the legacy mode or legacy workflow. Specifically this is the case when we are using ClusterLogging/ClusterLogForwarder named "instance" in "openshift-logging".  I have no idea how to make that succinct other then to call it "legacy" mode.

Reply:
Hopefully changes I've made are correct - I'm a little bit confused about when the RBAC / role bindings are required and when they are not, since I thought this procedure module is only required to support multi CLF.